app/models/work.rb

```diff
@@ -122,15 +122,17 @@ def default_work(title, user_id, collection_id, datacite_resource, ark)
   end
 
   def draft_doi
-    self.doi ||= "10.34770/tbd"
-    # TODO: Set up the doi to have  a variable prefix.  Test and production do not have the same one
-    # self.doi ||= begin
-    #                result = data_cite_connection.autogenerate_doi(prefix: "10.34770")
-    #                result.either(
-    #                   ->(response) { response.doi },
-    #                   ->(response) { raise("Something went wrong", response.status) }
-    #                 )
-    #              end
```

Comment:
The `either` syntax is what the datacite gem recommends (https://github.com/sul-dlss/datacite-ruby) is more elegant but does not seem to  really catch errors so I ditched it in favor of a good and tried if/else.